### PR TITLE
Map and Enum task

### PR DESCRIPTION
**Pull request by VaheAntonyan**

### CODE REVIEW

File: src/mapAndEnumTask/Program.java [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+import java.util.LinkedList;
+
+public class Program {
+    public static void main(String[] args) {
+        task1();
+        task2();
+    }
+
+    private static void task2() {
+        boolean leapYear = true;
```
Comment (DiannaAsatryan) [inline src/mapAndEnumTask/Program.java]: is it used?

File: src/mapAndEnumTask/Student.java [hunk 2]
```diff
@@ -0,0 +1,64 @@
+package mapAndEnumTask;
+
+import java.util.Objects;
+
+public class Student {
+    private String firstName;
+    private String lastName;
+    private int age;
+    private String phoneNumber;
+    private String faculty;
```
Comment (DiannaAsatryan) [inline src/mapAndEnumTask/Student.java]: I will suggest to use enum where possible, for example for faculty

File: src/mapAndEnumTask/Program.java [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+    private static void printFacultyAndStudentCount(HashMap<String, Integer> hashMap) {
+        for (Map.Entry<String, Integer> entry : hashMap.entrySet()) {
+            System.out.println(entry.getKey() + " " + entry.getValue());
+        }
+    }
+
+    private static HashMap<String, Integer> getStudentsCountByFacultyMap(List<Student> students) {
+        HashMap<String, Integer> hashMap = new HashMap<>();
+        for (Student st : students) {
+            hashMap.put(st.getFaculty(), hashMap.containsKey(st.getFaculty()) ? hashMap.get(st.getFaculty()) + 1 : 1);
```
Comment (DiannaAsatryan) [inline src/mapAndEnumTask/Program.java]: Then you can use EnumMap for this specific case: https://www.geeksforgeeks.org/enummap-class-java-example/

File: src/mapAndEnumTask/Month.java [hunk 4]
```diff
@@ -0,0 +1,44 @@
+package mapAndEnumTask;
+
+import mapAndEnumTask.myExceptions.DayException;
```
Comment (DiannaAsatryan) [inline src/mapAndEnumTask/Month.java]: don't put any upper case in package names

File: src/mapAndEnumTask/Program.java (hunk 1 above)
Comment (VaheAntonyan) [reply src/mapAndEnumTask/Program.java]: no, fixed

File: src/mapAndEnumTask/Student.java (hunk 2 above)
Comment (VaheAntonyan) [reply src/mapAndEnumTask/Student.java]: ok

File: src/mapAndEnumTask/Program.java (hunk 3 above)
Comment (VaheAntonyan) [reply src/mapAndEnumTask/Program.java]: ok

File: src/mapAndEnumTask/Month.java (hunk 4 above)
Comment (VaheAntonyan) [reply src/mapAndEnumTask/Month.java]: ok

**Comment by VaheAntonyan** [comment]
> Well done Vahe jan, just small comments to consider



> Well done Vahe jan, just small comments to consider

Thanks for review

**Pull request merged**